ui/app/AppLayouts/Profile/popups/BackupSeedModal.qml

```diff
@@ -19,6 +19,14 @@ StatusStackModal {
 
     property var privacyStore
 
+    onCurrentIndexChanged: {
+        if (currentIndex === 2) {
```

Comment:
My proposition is to put it directly to components:
```
BackupSeedStepBase {
            id: confirmFirstWord
            ...
            onVisibleChanged: if (visible) forceInputFocus()
}
```
to avoid comparing to index numbers and better fulfill solid's open-close principle.

Reply:
hey @micieslak visibility in StackLayout items is not working this way unless it's explicitly set like `visible: stack.currentIndex === 1`, `visible: stack.currentIndex === 2` etc . so it's kind of the same thing - the current index should be checked and compared. See video for credibility :D
https://user-images.githubusercontent.com/31625338/196982590-8daf01f8-011a-488a-835d-4c7352fe4116.mov

Reply:
Hey @alexandraB99. I found the source of misunderstanding :) `StackLayout` works exactly this way - current item is visible, the rest is not visible. But... we are not using raw `StackLayout`. We are using `StatusAnimatedStack` which is buggy and sets `visible` in a strange way, exactly as you presented (I was able to reproduce that as well). On this movie I print visibility after temporarily changing to plain `StackLayout`:

[Kazam_screencast_00071.webm](https://user-images.githubusercontent.com/20650004/197054551-ca170c38-fc3b-471b-899e-d7759b209f1e.webm)

Probably `StatusAnimatedStack` should have been based on `StackView` which has build-in support for animated transitions, but ofc it's a different story.
So I agree on current solution, but I would at least add some comment about the reason why it's done this way, and, ideally, create a new ticket for that bug in `StatusAnimatedStack`. Wdyt?

Reply:
@micieslak comment added and issue, there you go: https://github.com/status-im/status-desktop/issues/8024